app/javascript/react/screens/App/Overview/screens/MappingWizard/MappingWizard.js

```diff
@@ -1,66 +1,40 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import {
-  bindMethods,
-  noop,
-  Button,
-  Icon,
-  Modal,
-  Wizard
-} from 'patternfly-react';
+import { bindMethods, noop } from 'patternfly-react';
 import { createTransformationMappings } from './helpers';
-import MappingWizardBody from './MappingWizardBody';
-
-const mappingWizardSteps = [
-  'mappingWizardGeneralStep',
-  'mappingWizardClustersStep',
-  'mappingWizardDatastoresStep',
-  'mappingWizardNetworksStep',
-  'mappingWizardResultsStep'
-];
+import ModalWizard from '../../components/ModalWizard';
+import MappingWizardGeneralStep from '../MappingWizard/components/MappingWizardGeneralStep';
+import componentRegistry from '../../../../../../components/componentRegistry';
 
 class MappingWizard extends React.Component {
-  constructor() {
-    super();
-    this.state = { activeStepIndex: 0 };
-    bindMethods(this, ['prevStep', 'nextStep', 'goToStep']);
-  }
-
-  prevStep() {
-    const { activeStepIndex } = this.state;
-    this.setState({ activeStepIndex: Math.max(activeStepIndex - 1, 0) });
+  constructor(props) {
+    super(props);
+    const register = (context, component) => {
```

Comment:
if the idea is to make this a helper... i think it should go into helpers... also think we should name it something like `registerRegistryComponent` to indicate it is coming from the componentRegistry and we are binding it to `this`...

Reply:
Yeah, I wasn't sure if we needed this, just wanted to save a few lines. Also, this code could be cleaner if we just called the registry right in the step render functions, is there a reason we need it to assign things to `this`? I assumed so when I wrote it this way.